src/cli/services/linter.ts

```diff
@@ -23,6 +32,17 @@ export async function loadRulesets(cwd: string, rulesetFiles: string[]): Promise
   return readRuleset(rulesetFiles.map(file => (isAbsolute(file) ? file : resolve(cwd, file))));
 }
 
+const KNOWN_FORMATS: Array<[string, FormatLookup, string]> = [
+  ['oas2', isOpenApiv2, 'OpenAPI 2.0 (Swagger) detected'],
+  ['oas3', isOpenApiv3, 'OpenAPI 3.x detected'],
+  ['json-schema', isJSONSchema, 'JSON Schema detected'],
+  ['json-schema-loose', isJSONSchemaLoose, 'JSON Schema (loose) detected'],
+  ['json-schema-draft4', isJSONSchemaDraft4, 'JSON Schema Draft 4 detected'],
+  ['json-schema-draft6', isJSONSchemaDraft6, 'JSON Schema Draft 6 detected'],
+  ['json-schema-draft7', isJSONSchemaDraft7, 'JSON Schema Draft 7 detected'],
+  ['json-schema-2019-09', isJSONSchema2019_09, 'JSON Schema 2019-09 detected'],
```

Comment:
```suggestion
  ['json-schema-2019-09', isJSONSchema2019_09, 'JSON Schema Draft 2019-09 detected'],
```

Reply:
Is it a draft actually? I had `Draft` stated, but noticed they don't use 'Draft' word anymore.

Reply:
It is, only the versioning scheme changed, it is still a draft.